pages/RapPage.js

```diff
@@ -15,32 +17,44 @@ export default function RapCard() {
   if (!data) {
     return <h1> SORRY, no concerts for you! 🥀</h1>;
   }
+  const handleRapStarClick = (id) => {
+    router.push(`/rap/${id}`);
+  };
```

Comment:
You don't seem to be using this function.
Either delete it or find a use for it =)